app/views/albums/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="albums__topNav">
+    <%= button_to 'ADD NEW', new_album_path, { method: 'get', class: "albums__addButton" } %>
+</div>
+
+<table class="albums__table">
+  <tr class="albums__tableHeader">
+    <th><h3>Title</h3></th>
+    <th><h3>Artist</h3></th>
+    <th><h3>Details</h3></th>
+    <th><h3>Delete</h3></th>
+  </tr>
+  <% if @albums.length > 0 %>
+    <% @albums.each_with_index do |album, i| %>
+      <tr class=
+        <% if i % 2 == 0 %>
```

Comment:
but you should put this in to a Rails Helper method. https://signalvnoise.com/posts/1108-what-belongs-in-a-helper-method